curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/6632420f81f3cc554a5e540b.md

```diff
@@ -0,0 +1,770 @@
+---
+id: 6632420f81f3cc554a5e540b
+title: Step 65
+challengeType: 0
+dashedName: step-65
+---
+
+# --description--
+
+If you try to add a new task, edit that task, and then click on the `Add New Task` button, you will notice a bug.
+
+The form button will display the incorrect text of `"Update Task"` instead of `"Add Task"`. To fix this, you will need to assign the string `"Add Task"` to `addOrUpdateTaskBtn.innerText` inside your `addOrUpdateTask` function.
```

Comment:
It should be assigned to ```reset()``` function not ```addOrUpdateTask()```. 
It's currently in addOrUpdateTask() that is why we have this issue happening.

Reply:
Yes you are correct, I was just pushing that change.